<!DOCTYPE HTML>
<html>

    <head>
	<meta charset="utf-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
	<meta name="description" content="Davenport Lab at Penn State">
	<meta name="author" content="Emily Davenport">
	<meta name="generator" content="Hugo 0.154.3">
	<title>Publications &middot; Davenport Lab</title>
	<!-- Stylesheets -->
	
	<link rel="stylesheet" href="https://davenport-lab.github.io/css/main.css"/>
	
	

	
		<link rel="stylesheet" href="https://davenport-lab.github.io/css/academicons.min.css">
	


	<!-- Custom Fonts -->

	

	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.5.2/css/all.css">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.5.2/css/v4-shims.css">


	
	<link rel="shortcut icon" type="image/x-icon" href="https://davenport-lab.github.io/favicon.ico">
	<link rel="icon" type="image/x-icon" href="https://davenport-lab.github.io/favicon.ico">
	

	<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
	<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
	<!--[if lt IE 9]>
	<script src="js/ie/html5shiv.js"></script>
	<script src="js/ie/html5shiv.jsrespond.min.js"></script>
	<![endif]-->
</head>

    <body>

    <!-- Wrapper -->
        <div id="wrapper">

            <!-- Header -->
    <header id="header" class="alt">
        <a href="https://davenport-lab.github.io/" class="logo"><img src="/img/lab_logo_web.png" alt="" /></a>
        <nav>
            <a href="#menu">Menu</a>
        </nav>
    </header>

<!-- Menu -->
    <nav id="menu">
        <ul class="links">
            
                <li><a href="https://davenport-lab.github.io/">home</a></li>
            
                <li><a href="https://davenport-lab.github.io/research">research</a></li>
            
                <li><a href="https://davenport-lab.github.io/pages/people">people</a></li>
            
                <li><a href="https://davenport-lab.github.io/pages/openings">openings</a></li>
            
                <li><a href="https://davenport-lab.github.io/pages/publications">publications</a></li>
            
                <li><a href="https://davenport-lab.github.io/pages/contact">contact</a></li>
            

        </ul>
        <ul class="actions vertical">
            
            
        </ul>
    </nav>

        <!-- Main -->
            <div id="main" class="alt">

                
                    <section id="one">
                        <div class="inner">
                            <header class="major">
                                <h1>Publications</h1>
                            </header>
                            
                            <p>See my <a href="https://scholar.google.com/citations?user=bG50EJYAAAAJ&amp;hl=en">google scholar</a> profile for an up-to-date list.<br>
* denotes equal contribution<br>
<u>underlined</u> denotes Davenport lab member</p>
<h3 id="2025">2025</h3>
<ul>
<li>
<p>Van Syoc EP, Gomez A, <u>Davenport ER</u>, and Bordenstein SR. <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003390"><strong>Gut fungal profiles reveal phylosymbiosis and codiversification across humans and nonhuman primates</strong></a>. PLoS Biology 23(9):e3003390</p>
</li>
<li>
<p>Van Syoc EP, <u>Davenport ER</u>, and Bordenstein SR. <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003339"><strong>Gut fungi are associated with human genetic variation and disease risk</strong></a>. PLoS Biology 23(9):e3003339</p>
</li>
<li>
<p>Cappas V, Roy R, <u>Davenport ER</u>, and Sykes D. <a href="https://www.sciencedirect.com/science/article/pii/S1355030625000863"><strong>The microbiome and volatile organic compounds reflect the state of decomposition in an indoor environment</strong></a>. Science &amp; Justice. 2025;65(5):101302</p>
</li>
<li>
<p>Chilkina M, <u>Tran H</u>, Uribe G, McAninch S, McAninch C, Seideneck A, He B, Khanipov K, Golovko G, Powell DW, <u>Davenport ER</u>, Pinchuk IV. <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00324-9"><strong>MyD88-mediated signaling in intestinal fibroblasts regulates macrophage antimicrobial defense and prevents dysbiosis in the gut</strong></a>. Cell Reports. 2025;44(5):115553</p>
</li>
<li>
<p>Litichevskiy L, Considine M, Gill J, Shandar V, Cox TO, Descamps HC, Wright KM, Amses KR, Dohnalova L, Liou MJ, Tetlak M, Galindo-Fiallos MR, Wong AC, Lundgren P, Kim J, Uhr GT, Rahman RJ, Mason S, Merenstein C, Bushman FD, Raj A, Harding F, Chen Z, Prateek GV, Mullis M, Deighan AG, Robinson L, Tanes C, Bittinger K, Chakraborty M, Bhatt AS, Li H, Barnett I, <u>Davenport ER</u>, Broman KW, Cohen RL, Botstein D, Freund A, Di Francesco A, Churchill GA, Li M, and Thaiss CA. <a href="https://www.nature.com/articles/s41564-025-01963-3"><strong>Interactions between the gut microbiome, dietary restriction, and aging in genetically diverse mice</strong></a>. Nature Microbiology. 2025;10,1240-1257</p>
</li>
<li>
<p>Liu S, Rodriguez J, Munteanu V, Ronkowski C, Kumar Sharma N, Alser M, Andreace F, Blekhman R, Błaszczyk D, Chikhi R, Crandall KA, Libera KD, Francis D, Frolova A, Gancz AS, <u>Huntley NE</u>, Jaiswal P, Kosciolek T, Łabaj P, Łabaj W, Luan T, Mason C, Moustafa AM, Muralidharan HS, Mutlu O, Ghiasi NM, Rahnavard A, Sun F, Tian S, Tierney BT, Van Syoc E, Vicedomini R, Zackular JP, Zelikovsky A, Zielińska, Ganda E, <u>Davenport ER</u>, Pop M, Koslicki D, Mangul S. <a href="https://www.nature.com/articles/s43586-024-00376-6"><strong>Analysis of metagenomic data</strong></a>. Nature Reviews Methods Primers. 2025;5(5)</p>
</li>
</ul>
<h3 id="2024">2024</h3>
<ul>
<li>
<p>Chen YY, Evavold CL, Mann M, <u>Davenport ER</u>, McFall-Ngai M, Bienko M, Ueda HR, Tian L, Tjahjono N, Anikeeva P, Gogo Liu J, Deans TL, and Shen X. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0092867424008377"><strong>What tool or method do you wish existed?</strong></a> Cell. 2024;187(17)</p>
</li>
<li>
<p><u>Nichols RG</u> and <u>Davenport ER</u>. <a href="https://journals.asm.org/doi/full/10.1128/msystems.01480-24"><strong>Clade-specific long-read sequencing increases the accuracy and specificity of the gyrB phylogenetic marker gene</strong></a>. mSystems. 2024;10:e01480-24</p>
</li>
<li>
<p><u>Ryu EP</u>, Gautam Y, Proctor DM, Bhandari D, Tandukar S, <u>Gupta M</u>, Prasad Gautam G, Relman DA, Shibl AA, Bahadur Sherchand J, Jha AR, and <u>Davenport ER</u>. <a href="https://link.springer.com/article/10.1186/s40168-024-01941-7"><strong>Nepali oral microbiomes reflect a gradient of lifestyles from traditional to industrialized</strong></a>. Microbiome. 2024;12,228</p>
</li>
<li>
<p><u>Nichols RG</u>, Rimal B, Hao F, Peters JM, <u>Davenport ER</u>, and Patterson AD. <a href="https://www.sciencedirect.com/science/article/pii/S0160412024006081"><strong>Chlorpyrifos modulates the mouse gut microbiota and metabolic activity</strong></a>. Environment International. 2024;192:109022</p>
</li>
<li>
<p>Taye B, Mekonnen Z, Belanger KD, and <u>Davenport ER</u>. <a href="https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0012485"><strong>Gut-microbiome profiles among Soil-Transmitted Helminths (STHs) infected Ethiopian children enrolled in the school-based mass deworming program</strong></a>. PLoS Neglected Tropical Diseases 18(10), e0012485</p>
</li>
</ul>
<h3 id="2023">2023</h3>
<ul>
<li>
<p>Gancz AS, Farrer AG, Nixon M, Arriola L, Adler C, <u>Davenport ER</u>, Gully N, Cooper A, Britton K, Dobney K, Silverman J, Weyrich L. <a href="https://www.nature.com/articles/s41564-023-01527-3"><strong>Ancient dental calculus reveals novel oral microbiome diversity in pre-Industrialized British people</strong></a>. Nature Microbiology. 2023;8,2315-2325</p>
</li>
<li>
<p>Kelliher JM, Rudolph M, Vangay P, Abbas A, Borton MA, Santistevan C, <u>Davenport ER</u>, Davenport DW, Erazo NG, Herman C, Karstens L, Kocurek B, Lutz HL, Myers KS, Ockert I, Rodriguez FE, Saunders JK, Smith ML, Vogtmann E, Windsor A, Wood-Charlson EM, Woodley L, Eloe-Fadrosh EA. <a href="https://www.nature.com/articles/s41564-023-01361-7"><strong>Promoting cohort-based learning for community standards in microbiome research</strong></a>. Nature Microbiology. 2023;8, 751-753</p>
</li>
</ul>
<h3 id="2022">2022</h3>
<ul>
<li><u>Ryu EP</u> and <u>Davenport ER</u>. <a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-animal-020420-032054"><strong>Host genetic determinants of the microbiome across animals: From <i>Caenorhabditis elegans</i> to cattle</strong></a>. Annual Reviews Animal Biosciences. 2022;10, 203 - 226</li>
</ul>
<h3 id="2021">2021</h3>
<ul>
<li>Jeganathan NA, <u>Davenport ER</u>, Yochum GS, and Koltun WA. <a href="https://www.sciencedirect.com/science/article/pii/S2468867321000778"><strong>The microbiome in diverticulitis</strong></a>. Current Opinion in Physiology. 2021;22</li>
<li>Amato KR, Arrieta M, Azad MB, Bailey MT, Broussard JL, Bruggeling CE, Claud EC, Costello EK, <u>Davenport ER</u>, Dutilh BE, Swain Ewald HA, Ewald P, Hanlon EC, Julion W, Keshavarzian A, Maurice CF, Miller GE, Preidis GA, Segurel L, Singer B, Subramanian S, Zhao L, and Kuzawa CW. <a href="https://www.pnas.org/content/118/25/e2017947118.short"><strong>The human gut microbiome and health inequities</strong></a>. PNAS. 2021;118(25)</li>
<li>Marderstein AR, <u>Davenport ER</u>, Kulm S, Van Hout CV, Elemento O, Clark AG. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0002929720304341"><strong>Leveraging phenotyping variability to identify genetic interactions in human phenotypes</strong></a>. American Journal of Human Genetics. 2021;108(1)</li>
</ul>
<h3 id="2020">2020</h3>
<ul>
<li><u>Nichols RG</u>, <u> Davenport ER</u>. <a href="https://link.springer.com/article/10.1007/s00439-020-02237-0"><strong>The relationship between the gut microbiome and host gene expression: a review</strong></a>. Human Genetics</li>
<li><u>Davenport ER</u>. <a href="https://www.cell.com/trends/immunology/fulltext/S1471-4906(19)30242-X"><strong>Genetic Variation Shapes Murine Gut Microbiota via Immunity</strong></a>. Trends in Immunology. 2020;41(1)</li>
</ul>
<h3 id="2019">2019</h3>
<ul>
<li>Gong X, <u>Davenport ER</u>, Wang D, Clark AG. <a href="https://link.springer.com/article/10.1007/s10592-019-01146-8"><strong>Lack of spatial and temporal genetic structure of Anguilla japonica populations</strong></a>. Conservation Genetics. 2019</li>
<li>Kachoo P*, Eraso JM*, Beres SB, Olsen RJ, Zhu L, Nasser W, Bernard PE, Cantu CC., Ojeda Saavedtra M, José Arredondo M, Strope B, Do H, Kumaraswami M, Vuopio J, Gröndahl-Yli-Hannuksela K, Kristinsson KG, Gottfredsson M, Pesonen M, Pensar J, <u>Davenport ER</u>, Clark AG, Corander J, Caugant DA, Gaini S, Magnusses MD, Porter AR, DeLeoFR, and Musser JM. <a href="https://www.nature.com/articles/s41588-018-0343-1"><strong>Integrated analysis of population genomics, transcriptomics and virulence provides novel insights into Streptococcus pyogenes pathogenesis</strong></a>. Nature Genetics. 2019;51(3)</li>
</ul>
<h3 id="2018">2018</h3>
<ul>
<li>Jha AR, <u>Davenport ER</u>, Gautam Y, Bhandari D, Tandukar S, Ng K, Holmes S, Prasad Gautam G, Bahadur Sherchand J, Bustamante CD, and Sonnenburg JL. <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2005396"><strong>Gut microbiome transition across a life gradient in Himalaya</strong></a>. PLoS Biology. 2018;16(11):e2005396 (epub 2018 Nov 15)</li>
</ul>
<h3 id="2017">2017</h3>
<ul>
<li><u>Davenport ER</u>. <a href="https://www.sciencedirect.com/science/article/pii/S1931312817303517"><strong>Tooth be told, genetics influence oral microbiome</strong></a>. Cell Host &amp; Microbe. 2017;22(3)</li>
<li><u>Davenport ER</u>*, Sanders JG*, Song SJ, Amato KR, Clark AG, and Knight R. <a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-017-0454-7"><strong>The human microbiome in evolution</strong></a>. BMC Biology. 2017;15:127</li>
<li>Goodrich JK, <u>Davenport ER</u>, Clark AG, and Ley RE. <a href="http://www.annualreviews.org/doi/abs/10.1146/annurev-genet-110711-155532"><strong>The relationship between the human genome and microbiome comes into view</strong></a>. Annual Reviews Genetics. 2017;51 (epub head of print)</li>
<li><u>Davenport ER</u>. <a href="https://www.sciencedirect.com/science/article/pii/S1931312817303517"><strong>Tooth be told, genetics influences oral microbiome</strong></a>. Cell Host &amp; Microbe. 2017;22(3) [Commentary - not peer reviewed]</li>
<li>Igartua C, <u>Davenport ER</u>, Gilad Y, Nicolae DL, Pinto J, and Ober C. <a href="http://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-016-0227-5"><strong>Host genetic variation in mucosal immunity pathways influences the upper airway microbiome</strong></a>. Microbiome. 2017;5:16</li>
</ul>
<h3 id="2016">2016</h3>
<ul>
<li><u>Davenport ER</u>, Goodrich JK, Bell JT, Spector TD, Ley RE, and Clark AG. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-3290-1"><strong>ABO antigen and secretor statuses are not associated with gut microbiota composition in 1,500 twins</strong></a>. BMC Genomics. 2016;17:941</li>
<li>Beaumont M, Goodrich JK, Jackson MA, Yet I, <u>Davenport ER</u>, Vieira-Silva S, Debelius J, Pallister T, Mangino M, Raes J, Knight R, Clark AG, Ley RE, Spector TD, and Bell JT. <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1052-7"><strong>Heritable components of the human fecal microbiome are associated with visceral fat</strong></a>. Genome Biology. 2016;17(1):189</li>
<li>Goodrich JK, <u>Davenport ER</u>, Beaumont M, Jackson MA, Knight R, Ober C, Spector TD, Bell JT, Clark AG, and Ley RE. <a href="http://www.sciencedirect.com/science/article/pii/S1931312816301536"><strong>Genetic determinants of the gut microbiome in UK twins</strong></a>. Cell Host &amp; Microbe. 2016;19(5), 731-743</li>
<li>Goodrich JK*, <u>Davenport ER</u>*, Waters JL*, Clark AG, and Ley RE. <a href="http://science.sciencemag.org/content/352/6285/532"><strong>Cross-species comparisons of host genetic associations with the microbiome</strong></a>. Science. 2016;352(6285), 532-535</li>
<li><u> Davenport ER </u>.<a href="http://www.tandfonline.com/doi/full/10.1080/19490976.2016.1155022"><strong>Elucidating the role of the host genome in shaping microbiome composition</strong></a> Gut Microbes. 2016;7(2), 178-184</li>
<li>Blischak JD, <u>Davenport ER</u>, Wilson G. <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004668"><strong>A quick introduction to version control with Git and GitHub.</strong></a> PLoS Computational Biology. 2016;12(1):e1004668</li>
</ul>
<h3 id="2015">2015</h3>
<ul>
<li><u>Davenport ER</u>, Cusanovich DA, Michelini K, Barreiro LB, Ober C, and Gilad Y. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0140301"><strong>Genome-wide association studies of the human gut microbiota.</strong></a> PLoS One. 2015;10(11):e0140301</li>
</ul>
<h3 id="2014">2014</h3>
<ul>
<li>Zhou X, Cain CE, Myrthil M, Lewellen N, Michelini K, <u>Davenport ER</u>, Stephens M, Pritchard JK, and Gilad Y. <a href="#http://www.ncbi.nlm.nih.gov/pubmed/25468404"><strong>Epigenetic modifications are associated with inter-species gene expression in primates.</strong></a> Genome Biology. 2014; 15(12):547.</li>
<li><u>Davenport ER</u>, Mizrahi-Man O, Michelini K, Barreiro LB, Ober C, and Gilad Y. <a href="#http://www.ncbi.nlm.nih.gov/pubmed/24618913"><strong>Seasonal variation in human gut microbiome composition.</strong></a> PloS One. 2014;9(3):e90731</li>
</ul>
<h3 id="2013">2013</h3>
<ul>
<li>Mizrahi-Man O, <u>Davenport ER</u>, and Gilad Y. <a href="#http://www.ncbi.nlm.nih.gov/pubmed/23308262"><strong>Taxonomic classification of bacterial 16S rRNA genes using short sequencing reads: evaluation of effective study designs.</strong></a> PloS One. 2013;8(1):e53608</li>
</ul>

                        </div>
                    </section>
            <!-- Disqus Inject -->
                
            </div>
            
        <!-- Footer -->
            
                <!-- Footer -->
    <footer id="footer">
        <div class="inner">
            <ul class="icons">
                
                    <li><a href="https://www.github.com/davenport-lab" class="icon alt fa-github" target="_blank"><span class="label">GitHub</span></a></li>
                
                <a href="https://www.psu.edu/"><img src="/img/nittany_lion.jpg" alt="PSU logo" width="50" align="right"></a>

            </ul>
            <ul class="copyright">
                <li>&copy; Emily Davenport 2026</li>
                
            </ul>
        </div>
    </footer>

            
        </div>

    <!-- Scripts -->
        <!-- Scripts -->
    <!-- jQuery -->
    <script src="https://davenport-lab.github.io/js/jquery.min.js"></script>
    <script src="https://davenport-lab.github.io/js/jquery.scrolly.min.js"></script>
    <script src="https://davenport-lab.github.io/js/jquery.scrollex.min.js"></script>
    <script src="https://davenport-lab.github.io/js/skel.min.js"></script>
    <script src="https://davenport-lab.github.io/js/util.js"></script>

    

    <!-- Main JS -->
    <script src="https://davenport-lab.github.io/js/main.js"></script>

    

    

    </body>
</html>
